plaidml/edsl/edsl_test.cc

```diff
@@ -578,25 +578,25 @@ TEST(CppEdsl, LarsMomentum4d) {
 !f32 = type tensor<!eltwise.f32>
 module {
   func @lars_momentum4d(%arg0: tensor<4x7x3x9x!eltwise.f32>, %arg1: tensor<4x7x3x9x!eltwise.f32>, %arg2: !f32, %arg3: tensor<4x7x3x9x!eltwise.f32>) -> (tensor<4x7x3x9x!eltwise.f32>, tensor<4x7x3x9x!eltwise.f32>) {
-    %cst = "eltwise.sconst"() {value = 0.000000e+00 : f64} : () -> !f32
```

Comment:
In the long run, should probably fix this test to not expect a specific order of constants. But I don't think that's for this PR

Reply:
Yes, I'd like to fix this at some point...

If we could use lit for these tests that'd be ideal